### PR TITLE
Feature: file upload

### PR DESCRIPTION
**Pull request by ppezaris**

### CODE REVIEW

File: shared/agent/src/managers/serverManager.ts [hunk 1]
```diff
@@ -30,7 +30,21 @@ export class ServerManager {
 	@lspHandler(CodeStreamApiPostRequestType)
 	async post(request: { url: string; body?: any }): Promise<any> {
 		try {
-			return this.session.api.post(request.url, request.body);
+			let response;
```
Comment (bcanzanella) [inline shared/agent/src/managers/serverManager.ts:33]: test

Comment (bcanzanella) [inline shared/agent/src/managers/serverManager.ts:33]: a

**Comment by ppezaris** [comment]
testy test

File: shared/agent/package-lock.json [hunk 2]
```diff
@@ -3647,12 +3647,12 @@
...
-            "integrity": "sha512-WXieX3G/8side6VIqx44ablyULoGruSde5PNTxoUyo5CeyAMX6nVWUd0rgist/EuX655cjhUhTo1Fo3tRYqbcA==",
+            "version": "3.0.0",
+            "resolved": "https://registry.npmjs.org/form-data/-/form-data-3.0.0.tgz",
+            "integrity": "sha512-CKMFDglpbMi6PyN+brwB9Q/GOw0eAnsrEZDgcsH5Krhz5Od/haKHAX0NmQfha2zPPz0JpWzA7GJHGSnvCRLWsg==",
             "requires": {
                 "asynckit": "^0.4.0",
-                "combined-stream": "^1.0.6",
+                "combined-stream": "^1.0.8",
                 "mime-types": "^2.1.12"
             }
```
Comment (ppezaris) [inline shared/agent/package-lock.json:3657]: this is a review

**Review by ppezaris — COMMENTED** [review]
fooo

**Pull request merged**